src/transactions/TransactionUtils.cpp

```diff
@@ -1002,6 +1002,17 @@ getAsset(AccountID const& issuer, AssetCode const& assetCode)
     return asset;
 }
 
+void
+setClaimableBalanceClawbackEnabled(ClaimableBalanceEntry& cb)
+{
+    if (cb.ext.v() == 0)
```

Comment:
So I ended up changing the method and moving it to `CreateClaimableBalanceOpFrame`. I think it's specific enough to not require a `prepare`. This method won't be used outside of `CreateClaimableBalanceOpFrame`, so I didn't see any point in separating responsibilities. Please let me know if you disagree though.

Reply:
I think this is fine, because right now this is the only way to get `cb.ext.v() == 1`. If we add another flag, then I think we'll take the `prepare` approach.